packages/amplify-util-headless-input/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "build": "tsc",
-    "clean": "rimraf ./lib && rm tsconfig.tsbuildinfo",
+    "clean": "rimraf lib tsconfig.tsbuildinfo",
```

Comment:
Is rimraf a valid command in Windows?

Reply:
It's a node package. rm is the command that doesn't work on windows